tapdb/supply_commit_test.go

```diff
@@ -1760,7 +1764,9 @@ func TestSupplyCommitApplyStateTransition(t *testing.T) {
 	)
 
 	// Verify we have the extra one from before plus the new one.
-	precommitsRes = h.commitMachine.UnspentPrecommits(h.ctx, h.assetSpec)
+	precommitsRes = h.commitMachine.UnspentPrecommits(
+		h.ctx, h.assetSpec, false,
```

Comment:
Why shouldn't this be `true` here? This test is meant to exercise the workflow for updating the supply commitment for locally issued assets.